ditto/readers/synergi/db_parser.py

```diff
@@ -20,32 +19,150 @@ def __init__(self, input_file, **kwargs):
         self.SynergiDictionary = {}
         self.paths = {}
         self.paths["Synergi File"] = input_file
+        self.current_dir = os.path.realpath(os.path.dirname(__file__))
+        self.path_to_mdbtools = os.path.join(self.current_dir, "mdbtools/bin/")
+        self.TABLE_RE = re.compile(
+            "CREATE TABLE \[(\w+)\]\s+\((.*?\));", re.MULTILINE | re.DOTALL
+        )
+
+        self.DEF_RE = re.compile("\s*\[(\w+)\]\s*(.*?),")
 
         if "warehouse" in kwargs:
             self.paths["warehouse"] = kwargs["warehouse"]
 
         self.ParseSynergiDatabase()
 
+    # Using all the functions from the package, pandas_access and editing the path to the downloaded mdbtools path.
+
+    def list_tables(self, rdb_file, encoding="latin-1"):
```

Comment:
Can you make all this a separate file instead? Call it `pandas_access` and use it like you would if you imported a dependency. i.e.

```python
from . import pandas_access as pda
```

So in the future, we may be able to remove the file and use 

```python
import pandas_access as pda
```

instead.

Also, add `# Copyright (c) 2016 John Bjorn Nelson (tw: @generativist)` to the top of the new file named `pandas_access.py` and add a comment to below that with a link to the original repository and the reason for copy pasting the code.